packages/components/menu/src/use-menu-css-var.ts

```diff
@@ -0,0 +1,11 @@
+import useMenuColor from './use-menu-color'
+
+export const useMenuCssVar = (props) => {
+  return {
+    '--el-menu-text-color': props.textColor || '',
+    '--el-menu-hover-text-color': props.textColor || '',
+    '--el-menu-background-color': props.backgroundColor || '',
+    '--el-menu-hover-background-color': useMenuColor(props).value || '',
+    '--el-menu-active-color': props.activeTextColor || '',
+  }
+}
```

Comment:
**Question:** well, I think this method is confusing since it prefixed `useXXX` which is practically `hook` prefix but this method is not like any other `hooks` that invokes only inside `setup`, should we consider change the name?

Reply:
How about `generateMenuCssVar` ?

Reply:
Now, use `computed` for returned style.